src/i18n/es.json

```diff
@@ -1,4 +1,8 @@
 {
-    "app.title": "Edita <code>src/App.js</code> y guarda para recargar la página",
-    "app.link": "Aprende React"
-}
\ No newline at end of file
+  "app.title": "Edita <code>src/App.js</code> y guarda para recargar la página",
+  "app.link": "Aprende React",
+  "footer.allRightReserved" : "Todos los derechos reservados",
+  "footer.iso" : "Certificación de Calidad ISO 9001:2008",
+  "footer.privacy": "Políticas de privacidad y legales.",
+  "footer.privacy_link": "https://fromdoppler.com/privacidad"
```

Comment:
What do you think about calling it _url_ in place of _link_?

Reply:
Yes i can change. I copy it from the current footer